README.md

```diff
@@ -214,9 +213,11 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     let mut listener = event.listener().create()?;
 
-    while !SignalHandler::termination_requested() {
-        for event_id in listener.timed_wait(std::time::Duration::from_secs(1))? {
-            println!("event was triggered with id: {:?}", event_id);
+    while let ElkEvent::Tick = Elk::wait(Duration::ZERO) {
+        if let Ok(events) = listener.timed_wait(CYCLE_TIME) {
```

Comment:
This looks a bit odd. Maybe a convenience function like `Elk::whatchamacallit()` would be nice.